javascript-practice/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "my-project",
   "source": "src/pages/login.html",
   "scripts": {
-    "start": "parcel src/pages/login.html",
-    "build": "rm -rf .parcel-cache && parcel build src/pages/login.html"
+    "start": "parcel src/pages/dashboard.html",
```

Comment:
How can you run login page?

Reply:
I am getting an error in building parcel. I'm looking for a fix

Reply:
With your solution, we can not run login page. I can accept for now. But please resolve it soon. We can not demo without login page